lib/util.js

```diff
@@ -0,0 +1,4 @@
+exports.REGEX = {
+  // copied from http://stackoverflow.com/a/46181
+  email: /^([\w-]+(?:\.[\w-]+)*)@((?:[\w-]+\.)*\w[\w-]{0,66})\.([a-z]{2,6}(?:\.[a-z]{2})?)$/i
```

Comment:
Hmm, instead of exporting the email regex, maybe it's better to just export a function called `isValidEmail()`? That seems more readable and more extensible, since it allows us to add more non-regex-based code to the function in the future if we want. It's up to you though, this is ok as-is.